### ISSUE TITLE
Can't flash on n0100

### ISSUE THREAD

**Issue by RedGl0w** [zardam/nw-external-apps#4]
The link (
![image](https://user-images.githubusercontent.com/43498612/74054745-e0b8db80-49de-11ea-891f-580fb42d4bad.png))
make that
![image](https://user-images.githubusercontent.com/43498612/74054722-d4348300-49de-11ea-8a9d-5734255cd86c.png)

![image](https://user-images.githubusercontent.com/43498612/74054787-f4644200-49de-11ea-8cc7-82b7fa6d5bf9.png)

**Comment by zardam** [comment]
The website is not intended to work for a n0100, only for n0110, sorry.

**Comment by LeGmask** [comment]
yeah, but i think a better message would be perfect, because if some one doens't know his model that he could just think it's a bug

**Comment by RedGl0w** [comment]
> The website is not intended to work for a n0100, only for n0110, sorry.

The apps aren't for n0100
But you could specify than the firmware too
